sharktank/tests/models/llama/sharded_llama_test.py

```diff
@@ -18,6 +27,92 @@
 import torch
 from copy import deepcopy
 from shark_turbine.aot import FxProgramsBuilder, export
+import iree.runtime
+from pathlib import Path
+
+
+def get_iree_devices(device_count: int) -> List[iree.runtime.HalDevice]:
+    hal_driver = iree.runtime.get_driver("local-task")
```

Comment:
might be better to have the backend as an input arg instead of hardcoded "local-task" so that we can easily expand testing to add other backend support like rocm/hip.

Reply:
I added a parameter.